### PR TITLE
 Example BBRC

### PR DESCRIPTION
**Pull request by JEBROUN**
@mbaudin47

### CODE REVIEW

**Comment by JEBROUN** [comment]
@mbaudin47 
the RP53_Demonstration.ipynb and RP53_Demo.ipynb files are the same. I delete RP53_Demonstration.ipynb

**Comment by mbaudin47** [comment]
You added the new file "BBRC - EDF - Youssef.ipynb", which name contains blank spaces. This is not a intrinsic bug, but it makes the testing system go wrong: could you please rename it?

**Comment by JEBROUN** [comment]
@mbaudin47 
in these commits I changed the name of the file BBRC - EDF - Youssef.ipynb to benchmark_2019.ipynb
but, I find a problem with notebook and circle ci

**Comment by mbaudin47** [comment]
The test est trop long, c'est écrit :

```
+ Testing  ./examples/BBRC.ipynb
jupyter nbconvert --to notebook --execute BBRC.ipynb
[NbConvertApp] Converting notebook BBRC.ipynb to notebook
[NbConvertApp] Executing notebook with kernel: python3
[NbConvertApp] ERROR | Timeout waiting for execute reply (30s).
```

Quelle est la durée prévue d'exécution de ce NB?

File: examples/BBRC - EDF - Youssef.ipynb [hunk 1]
```diff
@@ -0,0 +1,746 @@
+{
```
Comment (mbaudin47) [inline examples/BBRC - EDF - Youssef.ipynb:1]: Please rename the file without blank spaces.

File: examples/BBRC2019.ipynb [hunk 2]
```diff
@@ -0,0 +1,596 @@
...
+    "We present the BBRC test cases using the otbenchmark module"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [
```
Comment (mbaudin47) [inline examples/BBRC2019.ipynb:46]: There should be no output here; some method probably generates an unwanted message here.

File: examples/BBRC2019.ipynb [hunk 3]
```diff
@@ -0,0 +1,596 @@
...
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We use the following methods to solve the optimization problem \"FORM\" and \"SORM\""
+   ]
+  },
+  {
```
Comment (mbaudin47) [inline examples/BBRC2019.ipynb:172]: This is an unwanted empty cell.

File: examples/BBRC2019.ipynb [hunk 4]
```diff
@@ -0,0 +1,596 @@
...
+    "the `compareMethode` method takes as a parameter a problem and it returns the probabilities estimated by each method, in addition, it returns the performance of these methods"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 13,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def CompareMethode(Problem):\n",
```
Comment (mbaudin47) [inline examples/BBRC2019.ipynb:393]: This is excellent. However, please translate the text into english.

File: examples/BBRC2019.ipynb [hunk 5]
```diff
@@ -0,0 +1,596 @@
...
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "graph1 = MonteCarloSampling(p57, 10000, 0.1)[2]\n",
+    "\n",
```
Comment (mbaudin47) [inline examples/BBRC2019.ipynb:554]: Only one single blank line is needed here.

File: examples/RP8_Demo.ipynb [hunk 6]
```diff
@@ -4,8 +4,8 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "# Problème RP8 \n",
-    "## Dans ce Notebook, nous traitons le problème RP8 de BBRC 2019 en utilisant la méthode FORM SORM et la méthode de Monte Carlo"
+    "# Probl�me RP8 \n",
```
Comment (mbaudin47) [inline examples/RP8_Demo.ipynb:7]: Please translate into english.

File: examples/scripts/main19_07.py [hunk 7]
```diff
@@ -10,13 +10,24 @@
 # ~ dist_X1 = ot.Normal(0., 1.)
 # ~ dist_X2 = ot.Normal(0., 1.)
 # ~ myDistribution = ot.ComposedDistribution([dist_X1, dist_X2])
+parameters1 = ot.LogNormalMuSigma(120, 12, 0.0)
```
Comment (mbaudin47) [inline examples/scripts/main19_07.py:13]: Perfect ;)

File: examples/BBRC - EDF - Youssef.ipynb (hunk 1 above)
Comment (JEBROUN) [reply examples/BBRC - EDF - Youssef.ipynb:1]: I changed this file to a file called benchmark_2019.ipynb !!!
you can find it in my last commit

**Comment by JEBROUN** [comment]
Hello @mbaudin47 !
I think the only problem is related to black-nb, it transforms notebooks so I can't access them

**Comment by mbaudin47** [comment]
I integrated the changes in the separate PR #34, so that the Notebooks format issue could be fixed. @JEBROUN : please let me know if this is OK for you. Then I will integrate PR #34 and drop PR #25. On the black-nb issue, please either use the patched version (with the bugfix included) or do not use the tool until the next release: this is because the current version crashes Notebooks with accents and make them unusable.

**Comment by mbaudin47** [comment]
The PR #34 integrates the same commits, with a few additional corrections. This is why I close this PR without integrating it.

**Pull request closed without merging**